pages/api/v1/child_bounties/createChildBounty.ts

```diff
@@ -0,0 +1,89 @@
+// Copyright 2019-2025 @polkassembly/polkassembly authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import { NextApiHandler } from 'next';
+import storeApiKeyUsage from '~src/api-middlewares/storeApiKeyUsage';
+import withErrorHandling from '~src/api-middlewares/withErrorHandling';
+import { isValidNetwork } from '~src/api-utils';
+import { postsByTypeRef } from '~src/api-utils/firestore_refs';
+import authServiceInstance from '~src/auth/auth';
+import { CreatePostResponseType } from '~src/auth/types';
+import getTokenFromReq from '~src/auth/utils/getTokenFromReq';
+import messages from '~src/auth/utils/messages';
+import { ProposalType } from '~src/global/proposalType';
+import { EAllowedCommentor, Post } from '~src/types';
+import getSubstrateAddress from '~src/util/getSubstrateAddress';
+
+const handler: NextApiHandler<CreatePostResponseType> = async (req, res) => {
+	storeApiKeyUsage(req);
+
+	try {
+		if (req.method !== 'POST') return res.status(405).json({ message: 'Invalid request method, POST required.' });
+
+		const network = String(req.headers['x-network']);
+		if (!network || !isValidNetwork(network)) return res.status(400).json({ message: messages.INVALID_NETWORK });
+
+		const { content, title, postId, proposerAddress, allowedCommentors, link, tags } = req.body;
+		if (!content || !title || !postId || !proposerAddress) return res.status(400).json({ message: messages.INVALID_PARAMS });
+
+		if (allowedCommentors && !Array.isArray(allowedCommentors)) {
```

Comment:
please check if items in allowedCommentors are valid members of EAllowedCommentor